### PR TITLE
Alexmz2/general edits

### PR DESCRIPTION
**Pull request by alex-m-zhang**
Fixed some text in ECE329 and ECE350 pages.
Changed placeholder text and renamed some files.

### CODE REVIEW

File: docs/Course Wiki/ECE Course Offerings/ECE329.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # ECE329
 
+ECE329 (Fields and Waves I) is a 3-credit-hour course that specifically required for EEs as part of the Electrical Engineering Core and satisfies the 1-of-6 Electrical Engineering Foundations Course requirement for CEs. This course also satisfies the 2-of-5 requirement for an EE minor. It is offered in the fall, spring, and summer.
```
Comment (rzig) [inline docs/Course Wiki/ECE Course Offerings/ECE329.md]: Grammar

File: docs/Course Wiki/ECE Course Offerings/ECE329.md [hunk 2]
```diff
@@ -28,7 +30,7 @@ The course overall starts off quite light, especially if the student is already
...
 
-The course will also have three midterms and a final. Difficulty of the midterms depends heavily on the professor who is organizing the course during that semester. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice.
+The course will also have three midterms and a final. Difficulty of the midterms depends heavily on the professor who is organizing the course during that semester. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice. Students are either provided a cheatsheet or allowed to bring index card(s) to the exam.
```
Comment (rzig) [inline docs/Course Wiki/ECE Course Offerings/ECE329.md]: How many index cards? Can I make the textbook into index cards

File: docs/Course Wiki/ECE Course Offerings/ECE350.md [hunk 3]
```diff
@@ -30,7 +32,7 @@ Take this course to satisfy 3 out of 5 credit for EEs or if you have an avid int
...
 
-The course will also have three midterms and a final. The midterms are generally not thought to be too difficult - more of your time will be spent on the homeworks. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice.
+The course will also have three midterms and a final. The midterms are generally not thought to be too difficult - more of your time will be spent on the homeworks. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice. Students are allowed to bring a cheatsheet (one 8.5x11 front and back), as well as cheatsheets from prior midterms. For example, for midterm 3 you would be allowed three sheets front and back.
```
Comment (rzig) [inline docs/Course Wiki/ECE Course Offerings/ECE350.md]: Clarify whether it has to be the *exact* same cheat sheet, or if the policy is midterm i you’re allowed i sheets.

**Review by rzig — CHANGES_REQUESTED** [review]

File: docs/Course Wiki/ECE Course Offerings/ECE329.md [hunk 4]
```diff
@@ -30,9 +30,9 @@ The course overall starts off quite light, especially if the student is already
...
+The course will also have three midterms and a final. Difficulty of the midterms depends heavily on the professor who is organizing the course during that semester. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice. Cheatsheet policies vary depending on the semester and the lead instructor. During some semesters, students are provided a cheatsheet. During others, students are allowed to bring an index card (3x5 inches) front and back to the exam, as well as index cards from prior midterms. For example, for midterm 3 you would be allowed three index cards front and back. You are highly recommended to reuse index cards from prior midterms, but this is not enforced.
```
Comment (rzig) [inline docs/Course Wiki/ECE Course Offerings/ECE329.md]: still unclear. do those three index cards have to be the exact ones you brought to prior midterms, or can they be any notecards. This should be worded either as:
1. "During others, students are allowed to bring one double-sided 3x5 index card to the first exam, two double-sided 3x5 index cards to the second exam, and so on"
OR
2. "During others, students are allowed to bring one double-sided 3x5 index card to each exam, along with the notecards they brought to any previous exams (note: these must be the *same* notecards, and does not mean that for the second midterm you can bring two notecards with only midterm 2 content)"
3.

File: docs/Course Wiki/ECE Course Offerings/ECE329.md [hunk 5]
```diff
@@ -30,9 +30,9 @@ The course overall starts off quite light, especially if the student is already
...
 
-Professor Ilie has been working to add a VR lab component to the class: see [ECE398RI2](ECE398.md). It is highly recommended to take this course concurrently with ECE329 as it allows you to see what the theoretical results you learn actually look like.
+Professor Ilie has been working to add a VR lab component to the class: see [ECE398RI2](ECE398.md). It is highly recommended to take this course concurrently with ECE329 as it allows you to literally see what the theoretical results you learn actually look like.
```
Comment (rzig) [inline docs/Course Wiki/ECE Course Offerings/ECE329.md]: rm literally

File: docs/Course Wiki/ECE Course Offerings/ECE350.md [hunk 6]
```diff
@@ -32,7 +32,7 @@ Take this course to satisfy 3 out of 5 credit for EEs or if you have an avid int
...
+The course will also have three midterms and a final. The midterms are generally not thought to be too difficult - more of your time will be spent on the homeworks. There are only a few practice exams - thus it is recommended to keep up with the homework every week. Professor Kudeki's notes have good practice questions with solutions in them, should the student find that they need more practice. Students are allowed to bring a cheatsheet (one 8.5x11 front and back), as well as cheatsheets from prior midterms. For example, for midterm 3 you would be allowed three sheets front and back. You are highly recommended to reuse cheatsheets from prior midterms, but this is not enforced.
```
Comment (rzig) [inline docs/Course Wiki/ECE Course Offerings/ECE350.md]: see wording comment above for how to make the policy more clear, last sentence here contradicts "as well as cheatsheets from prior exams"

**Review by rzig — CHANGES_REQUESTED** [review]

File: docs/Course Wiki/ECE Course Offerings/ECE329.md (hunk 5 above)
Comment (alex-m-zhang) [reply docs/Course Wiki/ECE Course Offerings/ECE329.md]: why :(
But fine.

**Review by rzig — APPROVED** [review]

**Pull request merged**